### PR TITLE
add lint file

### PR DESCRIPTION
**Pull request by Fobhep**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+jobs:
+  build:
+    env:
+      ANSIBLE_FORCE_COLOR: 'true'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
```
Comment (maximiliankolb) [inline .github/workflows/main.yml]: What's the reasoning behind running this with two different python versions?

Comment (Fobhep) [reply .github/workflows/main.yml]: copy paste from pulp.. :D
ie - none really, will remove it

**Comment by Fobhep** [comment]
@maximiliankolb let me know when you think this is done or rather just approve it - then I will merge it ;)

**Comment by maximiliankolb** [comment]
Looks good to me. 👍

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Pull request merged**